doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,139 @@
+.. procfs-plugin:
+
+======================================================================
+The ProcFS plugin
+======================================================================
+
+This feature is **tech preview** quality.
```

Comment:
How about making this a `note`, so it's more prominent?

Reply:
Done